test/package.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
```

Comment:
issue: There is already `package.test.js` file, this is risky since it can possibly cause name collisions. Please, rename the file(s) to, for example, `packageTypeScript.test.ts` or something similar.

Reply:
OK I understand, after transpilation to js the collision will happen.

Reply:
fixed